EnvironmentSetup/AWS/Source/Dockerfile

```diff
@@ -5,13 +5,16 @@ ENV TERM xterm-256color
 ENV COLORTERM truecolor
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
 ENV TF_DATA_DIR=/terraform-state
-RUN apt-get update && apt-get install awscli software-properties-common apt-transport-https locales curl tar git procps libncurses-dev -y && \
+RUN apt-get update && apt-get install software-properties-common apt-transport-https locales curl unzip tar git procps libncurses-dev -y && \
 	locale-gen en_US.utf8 && locale -a  && \
 	update-locale LANG=en_US.UTF-8 LANG=en_US.UTF8 LC_CTYPE=en_US.UTF8 LC_COLLATE=en_US.UTF8 && \
 	curl -LO "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl" && \
```

Comment:
I think we need to change this also. Here we are using older version of kubectl. Looks like latest version of all solved the issue.

Reply:
But the cluster version is v1.20 and there are many changes that come with v1.24. It may cause API version errors in objects.